tmva/sofie/inc/TMVA/ROperator_Range.hxx

```diff
@@ -0,0 +1,91 @@
+#ifndef TMVA_SOFIE_ROPERATOR_RANGE
+#define TMVA_SOFIE_ROPERATOR_RANGE
+
+#include "TMVA/SOFIE_common.hxx"
+#include "TMVA/ROperator.hxx"
+#include "TMVA/RModel.hxx"
+
+#include <sstream>
+
+namespace TMVA{
+namespace Experimental{
+namespace SOFIE{
+
+template <typename T>
+class ROperator_Range final : public ROperator
+{
+private:
+
+   std::string fNStart;
+   std::string fNLimit;
+   std::string fNDelta;
+   std::string fNOutput;
+   std::vector<Dim> fShape;
+   std::string fType;
+
+public:
+   ROperator_Range(){}
+
+   ROperator_Range(std::string start, std::string limit, std::string delta, std::string nameOutput):
+      fNStart(start), fNLimit(limit), fNDelta(delta),
+      fNOutput(UTILITY::Clean_name(nameOutput)) {
+      if (std::is_same<T, float>::value) {
```

Comment:
Can this be moved at compile time, e.g. with 
```
static_assert( (std::is_same_v<T, float> || std::is_same_v<T, Long64_t>) &&
                  ""TMVA::SOFIE - Unsupported type by Range operator"");
```
?

Reply:
note the usage of Long64_t: this is also preferable to int64 given how problematic that type is on different platforms.

Reply:
Thanks Danilo for the comments!
I don't agree here, `int64_t` is the standard signed integer with 64 bits and it is the right type to use it.